credoai/utils/dataset_utils.py

```diff
@@ -23,4 +28,50 @@ def concat_features_label_to_dataframe(X, y, sensitive_features):
         label_name = 'label'
         df[label_name] = sensitive_features
 
-    return df, sensitive_feature_name, label_name
\ No newline at end of file
+    return df, sensitive_feature_name, label_name
+
+class ColumnTransformerUtil:
```

Comment:
@amrasekh , here is the new code that gets feature names from the column transformer